simulation/traffic_simulator/include/traffic_simulator/entity/entity_base.hpp

```diff
@@ -146,6 +147,10 @@ class EntityBase
 
   virtual void requestAssignRoute(const std::vector<geometry_msgs::msg::Pose> & waypoints) = 0;
 
+  virtual void requestSpeedChange(
+    const double target_speed, const SpeedChangeTransition transition,
+    const SpeedChangeConstraint constaint, const bool continuous);
```

Comment:
`constaint` => `constraint`

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/648/commits/b608d5e2c700eea9af558ed92a2a6d2699994108